cli/command/environment_test.go

```diff
@@ -387,3 +387,90 @@ func TestUnlinkEnvironmentsInputErrors(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvironmentRecursiveDelete(t *testing.T) {
```

Comment:
👍 good test! Now you just need to merge from `api-refactor` and fix this up a bit to satisfy those changes.